.travis.yml

```diff
@@ -51,14 +51,21 @@ matrix:
         - PYTHON_VER='2.7'
 
 before_script:
+  - cd $HOME/Downloads
+  - mkdir -p eigen
+  - curl -Ls http://bitbucket.org/eigen/eigen/get/${Eigen_VERSION}.tar.gz | tar -xz -C eigen --strip-components=1
```

Comment:
I think you forgot to define `Eigen_VERSION` :grin:

Reply:
Thought it was pick up from `cmake` :stuck_out_tongue:  It failed on `cd $HOME/Downloads` though...